### PR TITLE
Refridgerated Tank Fix

### PR DESCRIPTION
**Pull request by Maleclypse**
#### Summary

SUMMARY: Bugfixes "Refridgerated Tank Fix"

#### Purpose of change

Contributes to #39546.   I've got it working as a tank. I've got it working as an electronic.  It may be refrigerating the spoilable milk.  I don't know because while it takes almost two days to rot in the vehicle menu, it a. never shows as cold, b. comes out fresh at least two days after putting it in when it's siphoned out and finally c. all contents disappear from the tank the second you take a single step, you can wait forever but if you debug change the time or walk a step everything inside vanishes.

#### Describe the solution

Witchcraft.  Flag witchcraft.  I've looked at the src and can't see any reason that it's acting weird but I speak C++ like I speak most languages.  I know a word or two and with a lot of diagrams I can maybe understand a sentence. 

#### Describe alternatives you've considered

Waking Cthulhu.

#### Testing

Tests are basically described up top.  I spawn in stuff I install I wait.

### CODE REVIEW

**Comment by RedShakespeare** [comment]
It may help to put a jug of milk aside as a reference when testing..?

**Comment by Maleclypse** [comment]
@anothersimulacrum do you have any ideas as to why these two flags are not interacting in the ways that it seems they should know that pockets work?

File: data/json/vehicleparts/tanks.json [hunk 1]
```diff
@@ -108,6 +108,45 @@
...
+    "size": 600500,
+    "pocket_data": [
+      {
+        "pocket_type": "CONTAINER",
+        "rigid": true,
+        "watertight": true,
+        "max_contains_volume": "60 L",
+        "max_contains_weight": "400 kg"
+      }
+    ],
```
Comment (anothersimulacrum) [inline data/json/vehicleparts/tanks.json:130]:
```suggestion
```

File: data/json/items/vehicle/utilities.json [hunk 2]
```diff
@@ -189,9 +189,18 @@
     "id": "fridgetank",
     "name": { "str": "refrigerated tank" },
     "description": "A 60L refrigerated tank for keeping liquids cool.  Provides some insulation from outside weather.",
-    "insulation": 2,
+    "insulation": 10,
     "volume": "60 L",
```
Comment (anothersimulacrum) [inline data/json/items/vehicle/utilities.json:193]: It just needs to be at least 1ml bigger, but figure it has more going on than just being a simple 60L tank.
```suggestion
    "volume": "72 L",
```

**Pull request merged**